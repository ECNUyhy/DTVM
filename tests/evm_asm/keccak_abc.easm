// keccak256("abc")
PUSH1 0x61
PUSH1 0x00
MSTORE8
PUSH1 0x62
PUSH1 0x01
MSTORE8
PUSH1 0x63
PUSH1 0x02
MSTORE8
PUSH1 0x03
PUSH1 0x00
KECCAK256
PUSH1 0x00
MSTORE
PUSH1 0x20
PUSH1 0x00
RETURN
