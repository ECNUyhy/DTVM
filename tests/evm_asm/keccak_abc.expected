status: success
error_code: 0
stack: []
memory: '4e03657aea45a94fc7d47ba826c8d667c0d1e6e33a64a036ec44f58fa12d6c45'
storage: {}
transient_storage: {}
return: '4E03657AEA45A94FC7D47BA826C8D667C0D1E6E33A64A036EC44F58FA12D6C45'
events: []
